src/decode.rs

```diff
@@ -4679,7 +4674,7 @@ pub(crate) unsafe fn rav1d_decode_frame_init_cdf(
         for (j, (ts, tile_start_off)) in iter::zip(
             slice::from_raw_parts_mut(f.ts, end + 1),
             if uses_2pass {
-                slice::from_raw_parts(f.frame_thread.tile_start_off, end + 1)
+                &f.frame_thread.tile_start_off[end + 1..]
```

Comment:
```suggestion
                &f.frame_thread.tile_start_off[..end + 1]
```

That said, I don't think we need the slice at all here.  The `zip` takes care of that.